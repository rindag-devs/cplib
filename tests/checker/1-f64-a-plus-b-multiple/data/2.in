5
1 2
1.2 3.4
-100 -200
123456.7 987654.321
-123456.7 -987654.321
