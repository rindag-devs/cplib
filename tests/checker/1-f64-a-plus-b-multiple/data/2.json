{"message":"Expected a float, got EOF","reader_trace_stacks":[{"fatal":true,"stack":[{"pos":{"byte":0,"col":0,"line":0},"var_name":"<ouf>"},{"pos":{"byte":0,"col":0,"line":0},"var_name":"output"},{"pos":{"byte":0,"col":0,"line":0},"var_name":"ans"},{"pos":{"byte":57,"col":0,"line":4},"var_name":"4"}]}],"score":0,"status":"wrong_answer"}
