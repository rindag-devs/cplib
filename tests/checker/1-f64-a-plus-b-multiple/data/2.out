3.000000000
4.600000000
-300.000000000
1111111.021000000
